### PR TITLE
Getting Started tutorial

### LINKED ISSUE #86: Getting started: let in PSCi
The "Getting Started" tutorial uses `let` twice in PSCi. This throws an error, as 
`Declarations in PSCi no longer require "let", as of version 0.11.0`

### PR DESCRIPTION
**Pull request by i3games**
Getting Started: removing let, adding type declarations (fixes #86 #87 #88) and in the test file.

### CODE REVIEW

File: guides/Getting-Started.md [hunk 1]
```diff
@@ -196,8 +199,10 @@ module Test.Main where
 import Prelude
 
 import Euler (answer)
-import Test.Assert (assert)
+import Control.Monad.Eff (Eff)
+import Test.Assert (assert, ASSERT)
 
+main :: forall e. Eff (assert :: ASSERT | e) Unit
```
Comment (paf31) [inline guides/Getting-Started.md]: I think this might be a bit confusing as part of a getting started guide. We could instead mention that the compiler will infer the type and we can choose to add it based on the warning. Most editor plugins provide a way to add the inferred type annotation.

Comment (i3games) [inline guides/Getting-Started.md]: I took them out again and wrote a paragraph explanation instead.

**Comment by hdgarrood** [comment]
I'd actually prefer we just put the type signatures in, perhaps with a comment that the reader shouldn't worry about precisely what they mean yet. In particular I imagine few will struggle with signatures for simpler types, e.g. `ns :: Array Int` or `answer :: Int` a little further up. We could also use a closed row for the `Eff` ones to make them slightly less intimidating. Whatever we do please let's not describe type signatures as an advanced topic.

**Comment by paf31** [comment]
I just worry that `Eff (console :: CONSOLE) Unit` is a complex type for a first-time user and it might put some users off. It's not an advanced topic, but it's certainly more advanced than `main = log "Hello World"`.

If we do include the type signature, I'd like to at least say something like "don't worry if you don't understand this type yet, the compiler will infer it anyway".

**Comment by paf31** [comment]
Ah I read the updated description. Yes, I agree we shouldn't say type annotations are an advanced topic. Maybe we can just reword it slightly.

**Comment by hdgarrood** [comment]
Ok, rewording it slightly sounds fine to me then. :+1:

**Comment by i3games** [comment]
Reworded it slightly.

File: guides/Getting-Started.md [hunk 2]
```diff
@@ -196,7 +198,7 @@ module Test.Main where
 import Prelude
 
 import Euler (answer)
-import Test.Assert (assert)
+import Test.Assert (assert, ASSERT)
```
Comment (hdgarrood) [inline guides/Getting-Started.md]: Let's revert the change to this line this since it's no longer necessary.

Comment (i3games) [inline guides/Getting-Started.md]: Done.

**Comment by hdgarrood** [comment]
Looks great, thank you!
